.pre-commit-hooks.yaml

```diff
@@ -183,3 +183,13 @@
     entry: trailing-whitespace-fixer
     language: python
     types: [text]
+-  id: update-commit-msg
+   name: update-commit-msg
+   entry: update-commit-msg
+   # Update to your ticket matching regex
+   args: ['--regex=TICKET-[0-9]']
```

Comment:
I'd probably just default this inside the hook itself, and leave out `args` here.

I'd probably default to `[A-Z]+-\d+`